pkg/config/config.go

```diff
@@ -620,6 +620,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("snmp_traps_enabled", false)
 	config.BindEnvAndSetDefault("snmp_traps_config.port", 162)
 	config.BindEnvAndSetDefault("snmp_traps_config.community_strings", []string{})
+	// No default as the agent falls back to `network_devices.namespace` if empty.
+	config.SetKnown("snmp_traps_config.namespace")
```

Comment:
Would it make sense to bind to an env var too, like other settings under `snmp_traps_config`?

Reply:
Yeah that's a good idea 👍